### ISSUE TITLE
Implement the profile manager

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#46]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

A profile manager class.

### Why

To separate lower-level ASP code into a dedicated class.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
